vhd/packer/install-dependencies.sh

```diff
@@ -66,9 +66,9 @@ installEtcd "docker"
 echo "  - etcd v${ETCD_VERSION}" >> ${VHD_LOGS_FILEPATH}
 
 VNET_CNI_VERSIONS="
-1.0.33
-1.0.30
-1.0.29
+1.1.3
+1.1.2
```

Comment:
If we know that 1.1.2 and 1.1.0 will break some scenarios do we still want to install them?